assignment/assignment7/stylesheet.scss

```diff
@@ -0,0 +1,144 @@
+@import 'reset';
+$gray_color:dimgray;
+@mixin set_width($value)
+{
+    width: $value;
+}
+.float_left
```

Comment:
I know I don't need to create class to just foat :right but I used it to access all sass features